src/Resources/config/page.php

```diff
@@ -133,8 +134,8 @@
         ->set('sonata.page.cms_manager_selector', CmsManagerSelector::class)
             ->public()
             ->tag('kernel.event_listener', [
-                'event' => 'security.interactive_login',
-                'method' => 'onSecurityInteractiveLogin',
+                'event' => LoginSuccessEvent::class,
+                'method' => 'onLoginSuccess',
             ])
```

Comment:
Same here, this is a BC break cause someone could dispatch manually the old event, what we can do is having both:
```
// NEXT_MAJOR: Stop listening 'security.interactive_login'.
->tag('kernel.event_listener', [
                'event' => 'security.interactive_login',
                'method' => 'onSecurityInteractiveLogin',
            ])
->tag('kernel.event_listener', [
                'event' => LoginSuccessEvent::class,
                'method' => 'onLoginSuccess',
            ])
```